### DISCUSSION TITLE
Initial password for connecting to module?

### DISCUSSION THREAD

**Discussion by markbellkosel** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#222] [Q&A] [answered]
I am very new to all this. I have everything set up but I must have missed the default log in info.

Edit: I found it!

**Comment by torei** [comment, marked as answer]
found in "src/config.h" i guess :-)
(just to put the answer to the question)

**Reply by kgreene1983** [reply]
Can it just be added to the install document
